src/DynamoCore/Graph/Nodes/DummyNode.cs

```diff
@@ -250,11 +251,10 @@ private void LoadNode(XmlNode nodeElement)
             OutputCount = Int32.Parse(outputCount.Value);
             LegacyNodeName = legacyName.Value;
 
-            if (nodeElement.ChildNodes != null)
+            foreach (XmlNode childNode in nodeElement.ChildNodes)
```

Comment:
if childNodes is null - this will throw an exception - I think you should keep the if

Reply:
https://stackoverflow.com/questions/3088147/why-does-net-foreach-loop-throw-nullrefexception-when-collection-is-null

Reply:
Resharper tells me that expression will always be true.